spec/lib/efile/nj/nj1040_calculator_spec.rb

```diff
@@ -279,7 +279,56 @@ def over_65_birth_year
       expect(instance.lines[:NJ1040_LINE_42].value).to eq(instance.lines[:NJ1040_LINE_39].value)
     end
   end
-  
+
+  describe 'line 64 - child and dependent care credit' do
+    let(:intake) { 
+      create(
+      :state_file_nj_intake,
+      :df_data_one_dep,
+      :fed_credit_for_child_and_dependent_care)
+    }
+    
+    context 'with an income of over 150k' do
+      it 'returns nil' do
+        allow(instance).to receive(:calculate_line_42).and_return 150_001
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_64].value).to eq(nil)
+      end
+    end
+
+    context 'with an income of 150k or less' do
+      it 'returns 10% of federal credit for an income of 150k or less' do
+        allow(instance).to receive(:calculate_line_42).and_return 150_000
+        instance.calculate
+        expect(instance.lines[:NJ1040_LINE_64].value).to eq(100)
+      end
+
+      it 'returns 20% of federal credit for an income of 120k or less' do
```

Comment:
[pebble] I think these would be more readable if we lost the "context 'with an income of 150k or less'" context and instead put each of these in their own context like 
```suggestion
context "with an income of 120k or less"
   let(:nj_taxable_income){ 120_000 }
      it 'returns 20% of federal credit' do
        expect(instance.lines[:NJ1040_LINE_64].value).to eq(200)
     end
end
```
with the following at the top of the describe block:
```
before do
        allow(instance).to receive(:calculate_line_42).and_return nj_taxable_income
        instance.calculate
end
```